grpcutil/dns_resolver.go

```diff
@@ -203,8 +208,13 @@ func (w *dnsWatcher) compileUpdate(newAddrs map[string]*Update) []*Update {
 }
 
 func (w *dnsWatcher) lookupSRV() map[string]*Update {
+	if w.service == "" {
+		level.Debug(w.logger).Log("msg", "not looking up DNS SRV record since w.service is empty")
```

Comment:
If empty string was passed, that was deliberate decision and we don't need to log about it.

Reply:
@pstibrany My reasoning for adding a _debug_ log line here is the library client may have made the mistake of passing an empty service parameter. Is it a problem to be a bit chatty on the debug level?

Reply:
I find it unnecessary, but it's no big deal.

Reply:
Yeah, whether there's any point to it or not is up for discussion.

Reply:
I also find this unnecessary. If we really want to keep it, then I would not reference `w.service` but have a more descriptive information, like "not looking up DNS SRV record because disabled".

Reply:
@pracucci @pstibrany removing the log message since it's no big deal to me, at least until it turns out to be useful, and consensus seems to be it's not necessary.